src/InputHandler.ts

```diff
@@ -2026,6 +2026,21 @@ export class InputHandler extends Disposable implements IInputHandler {
     }
   }
 
+  /**
+   * Helper to write color information packed with color mode.
+   */
+  private _updateColor(color: number, mode: number, c1: number, c2: number, c3: number): number {
```

Comment:
Little more specific?

```suggestion
  private _updateAttrColor(color: number, mode: number, c1: number, c2: number, c3: number): number {
```